src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/PrisonApiClientTest.kt

```diff
@@ -71,6 +71,19 @@ class PrisonApiClientTest {
     )
   }
 
+  @Test
+  fun `get user case loads fails`() {
+    HttpStatus.entries
+      .filter { it.is4xxClientError }
+      .map {
+        mockServer.stubGetUserCaseLoadsis4xxClientError(it.value())
+        assertThatThrownBy {
+          val userCaseLoads = prisonApiClient.getUserCaseLoads()
+          assertThat(userCaseLoads).isEmpty()
```

Comment:
do we get to the isEmpty? or is the exception thrown before that?

Reply:
it is same happened to me previous request. it doesn't reach that assertThat. corrected